### ISSUE TITLE
CLI: emqx_ctl rules create error: init_action_failure

### ISSUE THREAD

**Issue by dstmath** [emqx/emqx#3516]
**Environment**:

- EMQ X version (e.g. `emqx_ctl status`): 4.1.0
- Hardware configuration (e.g. `lscpu`):
```Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
Address sizes:       39 bits physical, 48 bits virtual
CPU(s):              8
On-line CPU(s) list: 0-7
Thread(s) per core:  2
Core(s) per socket:  4
Socket(s):           1
NUMA node(s):        1
Vendor ID:           GenuineIntel
CPU family:          6
Model:               60
Model name:          Intel(R) Core(TM) i7-4790 CPU @ 3.60GHz
Stepping:            3
CPU MHz:             1250.966
CPU max MHz:         4000.0000
CPU min MHz:         800.0000
BogoMIPS:            7183.14
Virtualization:      VT-x
L1d cache:           32K
L1i cache:           32K
L2 cache:            256K
L3 cache:            8192K
NUMA node0 CPU(s):   0-7
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt aes xsave avx f16c rdrand lahf_lm abm cpuid_fault epb invpcid_single pti tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid xsaveopt dtherm ida arat pln pts
```
- OS (e.g. `cat /etc/os-release`):`Debian GNU/Linux buster/sid`
- Kernel (e.g. `uname -a`):`Linux debian 4.19.0-4-amd64 #1 SMP Debian 4.19.28-2 (2019-03-15) x86_64 GNU/Linux`
- Erlang/OTP version (in case you build emqx from source code): 
- Others:

**What happened and what you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

```
/usr/bin/emqx_ctl resources create 'web_hook' -c '{"url": "https://***.***.com/mqtt/event", "method": "POST"}' -d 'test'
Resource resource:a49b29ec created

/usr/bin/emqx_ctl resources list
resource(id='resource:a49b29ec', type='web_hook', config=#{<<"method">> => <<"POST">>,<<"url">> => <<"https://hjbdev.komect.com/mqtt/event">>}, status=[#{is_alive => true,node => 'emqx@127.0.0.1'}], description='test')

/usr/bin/emqx_ctl rules create      "SELECT username as u, payload FROM \"message.publish\" where u='Steven'"      '[{"name":"data_to_webserver", "params": {"$resource":  "resource:a49b29ec"}}]'      -d "Forward publish msgs from steven to webserver"
Invalid options: {init_action_failure,[{{init_action_failure,'emqx@127.0.0.1'},{{emqx_web_hook_actions,on_action_create_data_to_webserver},badarg}}]}
```
emqx.log.1:
```
2020-06-05 10:58:33.447 [error] cluster_call error found, ResL: [{{init_action_failure,'emqx@127.0.0.1'},
                                  {{emqx_web_hook_actions,
                                       on_action_create_data_to_webserver},
                                   badarg}}
```

**Comment by HJianBo** [comment]
Hi, @dstmath I can't reproduce it. Did I miss some points?

First, I simulate an HTTP server:
```
while true; do echo -e "HTTP/1.1 200 OK\n\n $(date)" | nc -l 127.0.0.1 8080; done;
```

Then, I try to reproduce your problem in 4.1.0. It succeed:
```
 bin/emqx_ctl resources create 'web_hook' -c '{"url": "http://127.0.0.1:8080/mqtt/event", "method": "POST"}' -d 'test'
Resource resource:761bd091 created


bin/emqx_ctl rules create "SELECT username as u, payload FROM \"message.publish\" where u='Steven'"  '[{"name":"data_to_webserver", "params": {"$resource":"resource:761bd091"}}]' -d "Forward publish msgs from steven to webserver"
Rule rule:72f324e9 created
```

**Comment by HJianBo** [comment]
Sorry @dstmath It is a bug introduced v4.1.0. We'll fix it at v4.1.1


In the current, the workaround is adding the `payload_tmpl` field. E.g:
```
bin/emqx_ctl rules create "SELECT username as u, payload FROM \"message.publish\" where u='Steven'"  '[{"name":"data_to_webserver", "params": {"$resource":"resource:8ddb6580", "payload_tmpl": ""}}]' -d "Forward publish msgs from steven to webserver"
```

**Comment by terry-xiaoyu** [comment]
Fixed in emqx 4.1.1
